### PR TITLE
update help link and UG

### LINKED ISSUE #107: [PE-D][Tester A] User guide: Resume command
There are 2 address field within a resume command, which would not work if i use the same command as what they have said in the UG.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: YuSoonZ/ped#5

### LINKED ISSUE #110: [PE-D][Tester B] Format of date & time is not explained
In several of the commands, this time format: "tt/121220221400" is used without any explanation on the format.

I thought it was epoch time at first since it only consisted of numbers, and I was only able to derive what is potentially the format by seeing the example:

Even after thinking that the format is ddmmyyyyhhmm based on the UG, it was actually incorrect as trying to actually use that format would reveal that it was actually mmddyyyyhhmm instead. This wastes a lot of time for the users as they are forced to figure out the format on their own after many tries, making it very inconvenient to start using the basic functionalities of the app such as add.

It would be much better if the actual format was specifically stated in the UG, or better yet add dashes or spaces between the numbers, so that it is clear that these are date time

-------------
Labels: `severity.High` `type.DocumentationBug`
original: bryanyee33/ped#4

### LINKED ISSUE #136: [PE-D][Tester C] Help link not updated
This is the current help link. 

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: caitlyntang/ped#6

### LINKED ISSUE #138: [PE-D][Tester C] Leap year date error
Command `edit 2 tt/290220231200`. There is no 29 Feb 2023. By default java sets the date to 28 Feb 2023

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: caitlyntang/ped#7

### LINKED ISSUE #143: [PE-D][Tester D] Help command mistake
```
help
```

Output : https://se-education.org/addressbook-level3/UserGuide.html

Expected output : CCBot user guide

I believe you forget to update the help url to your own user guide. This might cause confusion to user since the url is not directing to your own user guide. 
[Honestly, it is easy to miss out :( ]

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: chiageng/ped#15

### PR DESCRIPTION
**Pull request by dabzpengu**
replace help link to team's UG page

add more explanation on the dateTime format and how leap days are handled

update constraints for year field

fix add resume formatting on UG

closes #143 
closes #136 
closes #110 
closes #138
closes #107

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -67,7 +67,21 @@ contact management tasks done faster than traditional GUI apps.
...
 
 --------------------------------------------------------------------------------------------------------------------
+## Date and Time
+> Your CCBot uses date and time to better help you manage your interviews. This section contains all the details on how
+> your CCBot interprets date and time.
+
+* ### Format : `ddMMyyyyHHmm`
+  * `dd` - refers to the day and the acceptable range is from **01** to **31**
+  * `MM` - refers to the month and the acceptable range is from **01** to **12**
+  * `yy` - refers to the year and accepts any **4-digit** numbers from **0001** to **9999**
```
Comment (Lalelulilulela) [inline docs/UserGuide.md]: I think you meant `yyyy` instead of `yy`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -402,18 +416,18 @@ This value cannot be modified or added by the `edit` or `add` commands.
...
+| `PRIORITY`           | Should be a number between 0 and 4.                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `INTERVIEW_TIME`     | Should be in the format `ddMMyyyyHHmm`.<br/>* `dd` - refers to the day and the acceptable range is from **01** to **31**<br/> * `MM` - refers to the month and the acceptable range is from **01** to **12**<br/>* `yy` - refers to the year and accepts any **4-digit** numbers from **0001** to **9999**<br/> * `HH` - refers to the hour and the acceptable range is from **00** to **23**<br/> * `mm` - refers to the minutes and the acceptable range is from **00** to **59** |
```
Comment (Lalelulilulela) [inline docs/UserGuide.md]: I think this could be added in the constraints of the AddCommand above as well

**Review by Lalelulilulela — COMMENTED** [review]
LGTM

**Comment by ZHANGTIANYAO1** [comment]
LGTM

**Pull request merged**